qiskit/circuit/library/__init__.py

```diff
@@ -127,6 +196,10 @@
 Basis Change Circuits
 =====================
 
+Basis change algorithms are :class:`~qiskit.circuit.library.BlueprintCircuit` subclasses
+that transform between bases. For example, in the case of the
+Quantum Fourier Transform (QFT), it transforms between the computational basis and the Fourier basis.
```

Comment:
They don't _have_ to be blueprints, QFT just happens to be one 🙂 
```suggestion
These circuits allow a basis transformation of the qubit states. For example, 
in the case of the Quantum Fourier Transform (QFT), it transforms between
the computational basis and the Fourier basis.
```